.travis.yml

```diff
@@ -20,6 +20,7 @@ install:
   - pip install -r requirements-test.txt
 
 before_script:
+  - chmod +x ./runflake8
```

Comment:
The root cause of this problem is that the permissions of these files are changed:

runflake8 100755 → 100644
runisort 100755 → 100644
runtests.py 100755 → 100644
tests/manage.py 100755 → 100644

Find a way to avoid changing those permissions and this problem won't happen, once you have done that, revert the changes to this file.